pkg/apis/zalando.org/v1/types.go

```diff
@@ -33,6 +33,10 @@ type ElasticsearchDataSetSpec struct {
 	// +optional
 	Replicas *int32 `json:"replicas,omitempty" protobuf:"varint,1,opt,name=replicas"`
 
+	// Exclude management of System Indices on this Data Set. Defaults to false
+	// +optional
+	ExcludeSystemIndices bool `json:"excludeSystemIndices"`
```

Comment:
FYI: You can use the option:

```go
// +kubebuilder:default:=true
```

In case you want to turn it into `IncludeSystemIndices` as discussed.

Reply:
thanks for the hint, I think I'll stick with excludeSystemIndices, it's kind of an "opt-out" and should be understanable by the users.